cmd/cireport/main.go

```diff
@@ -100,7 +101,16 @@ func main() {
 func init() {
 	flag.StringVar(&fullJobName, "job", "", "Full name of the test job (e.g. release-openshift-ocp-installer-e2e-openstack-serial-4.4)")
 	flag.StringVar(&jobIDs, "id", "", "Job IDs")
-	flag.StringVar(&username, "user", utils.GetUsername(), "Username to use for CI Cop")
+
+	flag.StringVar(&username, "user", os.Getenv("CIREPORT_USER"), "Username to use for CI Cop")
+	if username == "" {
+		if u, err := user.Current(); err == nil {
+			username = u.Username
+		}
+	}
+	if username == "" {
```

Comment:
This should be a `else` clause on line 109.

Reply:
I disagree and commit (`--amend`).